app/src/main/java/com/sunasterisk/fooddaily/utils/MutableListExt.kt

```diff
@@ -0,0 +1,29 @@
+package com.sunasterisk.fooddaily.utils
+
+import android.database.Cursor
+import com.sunasterisk.fooddaily.data.model.FoodDetail
+import com.sunasterisk.fooddaily.data.model.Instruction
+
+fun MutableList<FoodDetail>.addFoodDetail(cursor: Cursor) {
+    add(
+        FoodDetail(
+            cursor.getData(DatabaseKeys.FOOD_ID),
+            cursor.getData(DatabaseKeys.FOOD_TITLE),
+            cursor.getData(DatabaseKeys.FOOD_PRICE),
+            cursor.getData(DatabaseKeys.FOOD_READY_MINUTES),
+            cursor.getData(DatabaseKeys.FOOD_SUMMARY),
+            cursor.getData(DatabaseKeys.FOOD_IMAGE_URL)
+        )
```

Comment:
Tạo thêm 1 constructor nữa với tham số truyền vào là cursor trong model FoodDetail nhé

Reply:
hờ hờ, hqua Huy cmt là foods.addFoodDetail(it) nên t nghĩ đến việc dùng extension